problemsets-mat223/modules/module9.tex

```diff
@@ -71,7 +71,25 @@
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
 
-	XXX Finish
+	For $\mathcal S$:
+	
+	Let vectors $\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$ and scalar $\alpha$ be given. Then we need to verify that $\mathcal S(\vec u+\vec v)=\mathcal S(\vec u)+\mathcal S(\vec v)$ and $\mathcal S(a\vec u)=a\mathcal S(\vec u)$. Computing, we see:
+	\[
+	    \mathcal S(\vec u +\vec v)=\mathcal S\left(\mat{u_1+v_1\\u_2+v_2}\right)=\mat{2u_1+2v_1\\u_2+v_2}=\mat{2u_1\\u_2}+\mat{2v_1\\v_2}=\mathcal S(\vec u) + \mathcal S(\vec v)
+	\]
+	and
+	\[
+	    \mathcal S(a\vec u)=\mat{2a u_1\\a u_2}=\mathcal S\mat{a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal S(\vec u).
+	\]
+	Therefore, $\mathcal S$ is linear.
+	
+	For $\mathcal T$:
+	
+	Notice that $\mathcal T(\vec u+\vec v)$ doesn't look like $\mathcal S(\vec u)+\mathcal T(\vec v)$. Therefore, we will guess that $S$ is not linear and look for a counter example. Using $\xhat=\mat{1\\0}$ and $\yhat=\mat{0\\1}$, we see:
```

Comment:
Expand out what `\mathcal T(\vec u+\vec v)` and `\mathcal T(\vec u)+\mathcal T(\vec v)` is. You have `\mathcal S` mentioned as a typo...

Reply:
I have expanded out as you said. But I think that, if I expand out T(u+v) and T(u)+T(v), and show that they are not equal for all vectors, isn't that a proof for the question? Then the counter example looks duplicate for me